Assets/MixedRealityToolkit-Examples/UX/Scripts/DialogExamples.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.XR.WSA;
```

Comment:
breaks cross platform compilation

Reply:
Updated to use #ifdef following other existing code examples.